src/tiler/styles.mss

```diff
@@ -44,126 +51,195 @@
    Ensure that any changes to which stream_order + zoom levels are rendered
    here have appropriate coinciding filters in server.js
 */
+
+#drb_streams_50 {
+  line-color: @streamColor;
+  line-join: round;
+  line-cap: round;
+}
+
+#nhdflowline {
+  line-color: @streamColor;
+  line-join: round;
+  line-cap: round;
+}
+
+#nhd_quality_tp {
```

Comment:
These style declarations look to be repeated since you did the work of coalescing the values into groups in SQL.  You can render all three with one block if you join the selectors:
```css
#nhd_quality_tp, 
#nhd_quality_tn, 
#nhd_quality_tss {
   [nhd_qual_grp="L1"] {
     line-color: @wtrQualColor0;
   }
  ...
}